subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -158,6 +158,7 @@ Supported Package Managers:
 
 * https://asdf-vm.com/#/[Asdf-vm]
 * https://github.com/shyiko/jabba[Jabba]
+* https://maven.apache.org/guides/mini/guide-using-toolchains.html[Maven Toolchains]
```

Comment:
Maybe this belongs in its own section, since it's not really a package manager, like the others.

Reply:
I'll add another section for maven then. Is "Other supported formats" okay?

Reply:
How about:
```
JVM auto-detection knows how to work with:

* Operation-system specific locations (Linux, MacOs, Windows)
* Package Managers (https://asdf-vm.com/#/[Asdf-vm], https://github.com/shyiko/jabba[Jabba], https://sdkman.io/[SDKMAN!])
* https://maven.apache.org/guides/mini/guide-using-toolchains.html[Maven Toolchain] specifications
```